content/en/agent/autodiscovery/integrations.md

```diff
@@ -0,0 +1,543 @@
+* `<INTEGRATION_NAME>`: Name of the Datadog Integration
+* `<INIT_CONFIG>`: Configuration for the `init_config:` section for the given Datadog-`<INTEGRATION_NAME>`.
+* `<INSTANCE_CONFIG>`: Configuration for the `instances:` section for the given Datadog-`<INTEGRATION_NAME>`.
+
+Additionally if you use Agent v6.5+ you can also use the following parameter to configure your [Log collection][2] with Autodiscovery:
+
+* `<LOG_CONFIG>`: Configuration for the `logs:` section for the given Datadog-`<INTEGRATION_NAME>`
+
+Each tab in sections below shows a different way to apply Integration templates to a given container, available documented method are:
+
+* [Using a configuration file mounted within the Agent](?tab=file)
+* [Using Key-Value Store](?tab=keyvaluestore)
+* [Using Kubernetes Annotations](?tab=kubernetespodannotations)
+* [Using Docker Labels](?tab=dockerlabel)
+
+If you provide a template for the same Integration via multiple template sources, the Agent looks for templates in the following order (using the first one it finds):
+
+* Kubernetes annotations
+* Docker Labels
+* Files
+
+**Note**: Some supported integrations requires additional steps for Autodiscovery to work: [Ceph][3], [Varnish][4], [Postfix][5], [Cassandra Nodetools][6], and [Gunicorn][7]. Contact [Datadog support][8] for assistance.
+
+## Configuration
+
+{{< tabs >}}
+{{% tab "File" %}}
+
+Storing templates as local files and mounting them inside the containerized Agent doesn't require an external service or a specific orchestration platform. The downside is that you have to restart your Agent containers each time you change, add, or remove templates. The Agent looks for Autodiscovery templates in the mounted `/conf.d` directory, which contains default templates for the following checks: [Apache][1], [Consul][2], [CouchDB][3], [Couchbase][4], [Elasticsearch][5], [Etcd][6], [Kubernetes_state][7], [Kube_dns][8], [Kube_proxy][9], [Kyototycoon][10], [Memcached][11], [Redis][12], [Riak][13].
+
+Since v6.2.0 (and v5.24.0), the default templates use the default port for the monitored software, instead of auto-detecting it. If you need to use a different port, provide a custom Autodiscovery template either in [Docker container labels](?tab=docker-labels) or [Kubernetes pod annotations](?tab=kubernetes-annotations).
+
+These integration templates may suit you in basic cases, but if you need to customize your Datadog Integration configurations: such as enabling extra options, using different container identifiers, or using Template Variables indexing, you have to write your own auto-configuration files and use them with the Datadog Containerized Agent. To do this:
+
+1. Create a `autodiscovery.d/` folder on your host.
+2. Add your custom auto-configuration named `<INTEGRATION_NAME>.yaml`file to this folder.
+3. Mount this directory into the containerized Agent `conf.d/` folder. Note that on Kubernetes your can add them [using ConfigMaps][14].
+
+An alternative approach would be to:
```

Comment:
this is the correct approach, the AD templates should live in `conf.d/<INTEGRATION_NAME>.d/<CONF_NAME>.yaml`